### PR TITLE
feat: added wishlist feature

### PR DESCRIPTION
**Pull request by Vivek-Py**
## Wishlist
This allows user to add/remove items to/from wishlist.

## Deployment
https://bookmarkit-ecom-git-feat-wishlist-vivek-py.vercel.app/

### CODE REVIEW

File: src/utils/databaseQuery.js [hunk 1]
```diff
@@ -8,3 +8,36 @@ export async function getAllProducts() {
     return products;
   }
 }
+
+export async function getAllWishlistProducts() {
```
Comment (aurobindoGupta) [inline src/utils/databaseQuery.js:12]: noice work bhai, backend selfmade.

**Review by aurobindoGupta — APPROVED** [review]
nice work excellent code, good going!!
clear distinct components.

File: src/utils/databaseQuery.js [hunk 2]
```diff
@@ -8,3 +8,36 @@ export async function getAllProducts() {
...
+export async function addToWishlist(args) {
+  const { data: added, error } = await supabase.from("wishlist").insert([args]);
+  if (error) {
+    return [];
+  } else {
+    return added;
+  }
+}
+
+export async function removeFromWishlist(args) {
```
Comment (deepak29-git) [inline src/utils/databaseQuery.js:33]: variable name is good

**Review by deepak29-git — APPROVED** [review]

File: src/utils/databaseQuery.js (hunk 1 above)
Comment (Vivek-Py) [reply src/utils/databaseQuery.js:12]: Thanks auro!

File: src/utils/databaseQuery.js (hunk 2 above)
Comment (Vivek-Py) [reply src/utils/databaseQuery.js:33]: Thanks deepak!

**Pull request merged**